app/src/main/java/org/mozilla/tv/firefox/experiments/ExperimentsProvider.kt

```diff
@@ -44,6 +44,20 @@ class ExperimentsProvider(private val fretboard: Fretboard, private val context:
         }
     }
 
+    fun shouldShowHintBar(): Boolean {
+        val expDescriptor = checkBranchVariants(ExperimentConfig.HINT_BAR_TEST)
+        return when {
+            // The user is currently not part of the experiment
+            expDescriptor == null -> false
```

Comment:
Sentry.capture(NotInExperimentException("HintBarTest"))?

Reply:
I'm not totally clear on the meaning of a null descriptor as opposed to our else branch.  Where could I look to find that?  I wonder if it might be valuable to refactor `checkBranchVariants` to return a non-null type that falls back on a descriptive name (e.g., something like `ExperimentDescriptor.UserNotInExperiment`).